mcproto/types.go

```diff
@@ -106,6 +108,11 @@ type LegacyServerListPing struct {
 	ServerPort      uint16
 }
 
+// PingPayload represents the status ping payload (packet 0x01)
+type PingPayload struct {
+	Value int64
```

Comment:
This field name should be `Timestamp`, right? According to

https://minecraft.wiki/w/Java_Edition_protocol/Packets#Ping_Request_(status)